### PR TITLE
Remove admin links in notification emails

### PR DESCRIPTION
**Pull request by jmyatt**

### CODE REVIEW

**Review by Dillon-Dumesnil — APPROVED** [review]
Looks fine. I might recommend having Mike Terry take a look just since he has a little more context on the emails and can better confirm if these are all the spots.

**Review by mikix — APPROVED** [review]
You might be able to also drop calculating/passing-in the `course_approval_url` template variable now, if it's no longer used.

File: course_discovery/apps/course_metadata/emails.py [hunk 1]
```diff
@@ -112,7 +112,6 @@ def send_email(course_run, template_name, subject, to_users, recipient_name, con
         'org_name': org.name,
         'contact_us_email': project_coordinator.email,
         'course_page_url': review_url,
-        'course_approval_url': approve_url,
```
Comment (mikix) [inline course_discovery/apps/course_metadata/emails.py:115]: You can also drop the calculation of approve_url above this.

Comment (jmyatt) [reply course_discovery/apps/course_metadata/emails.py:115]: d'oh!  thanks

**Pull request merged**